resources/views/network-data/dashboard.blade.php

```diff
@@ -36,9 +36,26 @@
             {!! HTML::panelOpen('My Pilot Sessions', ['type' => 'vuk', 'key' => 'letter-p']) !!}
             <div class="row">
                 <div class="col-md-12" style="margin-bottom: 15px;">
-                    <p class="text-center">
-                        Coming soon!
-                    </p>
+                    <span style="display: flex; justify-content: center;">{{ $pilotSessions->appends('atcSessions', Input::get('atcSessions'))->links() }}</span>
+                    <table class="table table-striped tabled-bordered table-hover">
+                        <tr>
+                            <th>Callsign</th>
+                            <th>Departure</th>
+                            <th>Arrival</th>
+                            <th>Duration</th>
+                            <th>Logged On</th>
+                        </tr>
+                        @foreach($pilotSessions as $pilot)
+                            <tr>
+                                <td>{{ $pilot->callsign }}</td>
+                                <td>{{ $pilot->departure_airport }}</td>
+                                <td>{{ $pilot->arrival_airport }}</td>
+                                <td>{{ $pilot->human_duration }}</td>
+                                <td>{{ HTML::fuzzyDate($pilot->connected_at) }}</td>
```

Comment:
When looking at the history page, fuzzy dates are actually quite annoying to the end user - the entire purpose of the page is to provide information/data, so we should display straight off what the date and time was (this especially applies when things get grouped by month and year - you don't want to have to hover over every entry just to find out what day a session was).